src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -492,25 +517,26 @@ InferenceProfiler::InferenceProfiler(
 
 cb::Error
 InferenceProfiler::Profile(
-    const size_t concurrent_request_count, std::vector<PerfStatus>& summary,
-    bool& meets_threshold, bool& is_stable)
+    const size_t concurrent_request_count,
+    std::vector<PerfStatus>& perf_status_vec, bool& meets_threshold,
```

Comment:
naming the type (vec) is generally a bad idea. How about perf_statuses?

Reply:
I have iterated on these names a few times already because they were not clear. I agree that including a hint at the underlying data structure is not ideal so I will update it to `perf_statuses`